src/EnergyPlus/DaylightingManager.cc

```diff
@@ -9999,14 +9998,26 @@ void ReportIllumMap(EnergyPlusData &state, int const MapNum)
         SavedMnDy(MapNum) = state.dataEnvrn->CurMnDyHr.substr(0, 5);
     }
     if (EnvrnPrint(MapNum)) {
+        std::string refPt2;
+        if (state.dataDaylightingData->ZoneDaylight(state.dataDaylightingData->IllumMap(MapNum).Zone).TotalDaylRefPoints > 1) {
+            refPt2 = "";
+            for (int i = 2; i <= state.dataDaylightingData->ZoneDaylight(state.dataDaylightingData->IllumMap(MapNum).Zone).TotalDaylRefPoints; ++i) {
+                refPt2 += RefPts(state.dataDaylightingData->IllumMap(MapNum).Zone, i);
+                if (i < state.dataDaylightingData->ZoneDaylight(state.dataDaylightingData->IllumMap(MapNum).Zone).TotalDaylRefPoints) {
+                    refPt2 += ",";
+                }
+            }
+        } else {
+            refPt2 = format("RefPt{}=({:.2R}:{:.2R}:{:.2R})", 0, 0.00, 0.00, 0.00);
+        }
         WriteDaylightMapTitle(state,
                               MapNum,
                               *state.dataDaylightingData->IllumMap(MapNum).mapFile,
                               state.dataDaylightingData->IllumMap(MapNum).Name,
                               state.dataEnvrn->EnvironmentName,
                               state.dataDaylightingData->IllumMap(MapNum).Zone,
                               RefPts(state.dataDaylightingData->IllumMap(MapNum).Zone, 1),
-                              RefPts(state.dataDaylightingData->IllumMap(MapNum).Zone, 2),
+                              refPt2,
```

Comment:
This could be a single parameter. If I'm following correctly, these just get concatenated anyway.